voorhoede/tudelft-spacefinder
# Feat/remove stationary pc filter

# Changes


[feat: remove stationary pc filter](https://github.com/voorhoede/tudelft-spacefinder/commit/66a2087a472d45659d4599b97aed7cb73b015969)

# Associated issue


Resolves: https://trello.com/c/22WxOUOm/106-remove-losse-pc-filter

# How to test


1. Open preview link
2. Check if the pc filters is gone and all symbols.

# Checklist
